securestoragelibrary/src/main/java/de/adorsys/android/securestoragelibrary/SecurePreferences.java

```diff
@@ -84,14 +97,29 @@ public static float getFloatValue(@NonNull String key, @NonNull Context context,
         return Float.parseFloat(getStringValue(key, context, String.valueOf(defValue)));
     }
 
-    public static float getLongValue(@NonNull String key, @NonNull Context context, long defValue) {
+    public static long getLongValue(@NonNull String key, @NonNull Context context, long defValue) {
         return Long.parseLong(getStringValue(key, context, String.valueOf(defValue)));
     }
 
-    public static float getIntValue(@NonNull String key, @NonNull Context context, int defValue) {
+    public static int getIntValue(@NonNull String key, @NonNull Context context, int defValue) {
         return Integer.parseInt(getStringValue(key, context, String.valueOf(defValue)));
     }
 
+    public static Set<String> getStringSetValue(@NonNull String key, @NonNull Context context, Set<String> defValue) {
```

Comment:
add @NonNull annotation for the Set, please
add @NonNull for the return value as well